### ISSUE TITLE
Failure to load OS: AMFI and macOS public beta 10.15.4b1

### ISSUE THREAD

**Issue by ts1g** [osy86/HaC-Mini#187]
After installing the public 10.15.4b1, the machine refuses to boot beyond an initial logo splash screen. Resetting NVRAM and reinstalling the EFI with the newest HaC-Mini 2.6 yield the same results. 

In reading about the new AMFI requirement in 10.15.4, it appears that AMFI is a step up in security from SIP. Is the process for disabling AMFI similar to that of SIP a few versions back?

**Comment by osy86** [comment]
Can you enable verbose boot and take a picture of where it gets stuck?

**Comment by ts1g** [comment]
I just got back a round to this, after leaving the machine unplugged for a few days while on a trip, it booted up normally, and I was able to load public beta 15.4b2 with no issue. After a few hours, though, the machine locks up and requires power to be pulled for an hour (I the keyboard plugged into the front yellow USB, with the BIOS setting you recommend so that the machine can go to sleep) to allow machine to boot properly. I'm going to close this, since it doesn't seem to be an 15.4 beta issue, and then open a report on the lockup.
